printing/printer.ml

```diff
@@ -34,6 +34,49 @@ let delayed_emacs_cmd s =
 let get_current_context () =
   Pfedit.get_current_context ()
 
+let enable_unfocussed_goal_printing = ref false
+let enable_goal_tags_printing = ref false
+let enable_goal_names_printing = ref false
+
+let should_tag() = !enable_goal_tags_printing
+let should_unfoc() = !enable_unfocussed_goal_printing
+let should_gname() = !enable_unfocussed_goal_printing
+
+
+let _ =
+  let open Goptions in
+  declare_bool_option
+    { optsync  = false;
```

Comment:
But we really need to have both ways of setting options: synchronized and unsynchronized.